Documentation/gettingstarted/encryption-ipsec.rst

```diff
@@ -265,25 +283,65 @@ Troubleshooting
-                anti-replay context: seq 0x0, oseq 0xe0c0, bitmap 0x00000000
-                sel src 0.0.0.0/0 dst 0.0.0.0/0
-        src 10.60.1.1 dst 10.60.0.1
-                proto esp spi 0x00000001 reqid 1 mode tunnel
-                replay-window 0
-                auth-trunc hmac(sha256) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546 96
-                enc cbc(aes) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546
-                anti-replay context: seq 0x0, oseq 0x0, bitmap 0x00000000
-                sel src 0.0.0.0/0 dst 0.0.0.0/0
+   * XFRM stats with state:
+
+    Check if the packets count increases as you send traffic.
+
+    Following is the output from the source node.
+
+    .. code-block:: shell-session
+
+       $ ip -s xfrm s
+       src 10.60.0.1 dst 10.60.1.1
+               proto esp spi 0x00000001 reqid 1 mode tunnel
+               replay-window 0
+               auth-trunc hmac(sha256) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546 96
+               enc cbc(aes) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546
+               anti-replay context: seq 0x0, oseq 0xe0c0, bitmap 0x00000000
+               sel src 0.0.0.0/0 dst 0.0.0.0/0
+               lifetime config:
+                 limit: soft (INF)(bytes), hard (INF)(bytes)
+                 limit: soft (INF)(packets), hard (INF)(packets)
+                 expire add: soft 0(sec), hard 0(sec)
+                 expire use: soft 0(sec), hard 0(sec)
+               lifetime current:
+                 9507(bytes), 137(packets)
+                 add 2021-02-10 08:20:09 use 2021-02-10 08:30:12
+               stats:
+                 replay-window 0 replay 0 failed 0
+
+    Following is the output from the destination node.
+
+    .. code-block:: shell-session
+
+       $ ip -s xfrm s
+       src 10.60.1.1 dst 10.60.0.1
+               proto esp spi 0x00000001 reqid 1 mode tunnel
+               replay-window 0
+               auth-trunc hmac(sha256) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546 96
+               enc cbc(aes) 0x6162636465666768696a6b6c6d6e6f70717273747576777a797a414243444546
+               anti-replay context: seq 0x0, oseq 0xe0c0, bitmap 0x00000000
+               sel src 0.0.0.0/0 dst 0.0.0.0/0
+               lifetime config:
+                 limit: soft (INF)(bytes), hard (INF)(bytes)
+                 limit: soft (INF)(packets), hard (INF)(packets)
+                 expire add: soft 0(sec), hard 0(sec)
+                 expire use: soft 0(sec), hard 0(sec)
+               lifetime current:
+                 9507(bytes), 137(packets)
+                 add 2021-02-10 08:20:09 use 2021-02-10 08:30:12
+               stats:
+                 replay-window 0 replay 0 failed 0
+
+   * BPF program to decrypt traffic:
+
+    Check if the BPF programs to decrypt traffic is attached to relevant interfaces.
```

Comment:
How does this read? AFAIR, we now attach the program to all except reserved interfaces (e.g., veth, etc)?

Reply:
I think so. But yeah the reader's question then becomes "Which interfaces are relevant?" . Maybe 'all network-facing interfaces or alternatively all interfaces matching the configuration of `--encrypt-interface`'?

Also one nit, I think it should be "programs ... are" or "program ... is":
```suggestion
    Check if the BPF program to decrypt traffic is attached to relevant interfaces.
```

Reply:
oh and I wouldn't suggest mentioning `eth0` since some common distros like Ubuntu won't create such an interface (as it uses the new alphabet soup naming scheme).